function-app/adb-to-purview/src/Function.Domain/Helpers/PurviewCustomType.cs

```diff
@@ -632,9 +649,15 @@ public async Task<bool> Delete_Unused_Entity(string quilifiedName, string typeNa
         {
             var correlationId = Guid.NewGuid().ToString();
             var token = this.GetToken();
-            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.purview.azure.com/catalog/api";
-            var purviewSearchEndpoint = $"{baseurl}/atlas/v2/entity/bulk/uniqueAttribute/type/";
-            var deleteEndPoint = $"{baseurl}/atlas/v2/entity/guid/";
+            var baseurl = $"https://{Environment.GetEnvironmentVariable("PurviewAccountName") ?? ""}.{Environment.GetEnvironmentVariable("ResourceUri") ?? "purview.azure.com"}";
```

Comment:
This breaks anyone's existing appsettings since the current default is https://purview.azure.com

Reply:
We will need to update the settings, or can introduce a new one, what you think it is best?

Reply:
Per our conversation, we should look into changing the environment variable from `ResourceUri` to a different name and then include in the release notes how `ResourceUri` is no longer necessary in App Settings.